static/scss/dashboard.scss

```diff
@@ -490,6 +490,20 @@
       > p {
         color: $font-black;
       }
+
+      .tos-container {
+        max-height: 200px;
+        overflow-x: hidden;
+        overflow-y: auto;
```

Comment:
can you do `overflow-y: scroll` instead of auto? just captures our intent better (the styling is more self-documenting that way).